napari_allencell_annotator/view/viewer.py

```diff
@@ -137,3 +151,30 @@ def get_all_point_annotations(self) -> dict[str, list[tuple]]:
             all_point_annotations[points_layer.name] = self.get_selected_points(points_layer)
 
         return all_point_annotations
+
+    def toggle_points_layer(self, annot_points_layer: Points) -> None:
+        """
+        If the points layer mode is pan_zoom, set it to add, change other layers to pan_zoom, and select the current
+        points layer. Otherwise, unselected the latest point and set the mode to pan_zoom.
+
+        Parameters
+        ----------
+        annot_points_layer: Points
+            The target points layer
+        """
+        if self.get_points_layer_mode(annot_points_layer) == PointsLayerMode.PAN_ZOOM.value:
+            self.set_all_points_layer_to_pan_zoom()
+            self.set_points_layer_mode(annot_points_layer, PointsLayerMode.ADD)
+            self.viewer.layers.selection.clear()
+            self.viewer.layers.selection.add(annot_points_layer)
+        else:
+            annot_points_layer.selected_data = []
+            self.set_points_layer_mode(annot_points_layer, PointsLayerMode.PAN_ZOOM)
```

Comment:
comments

Reply:
added in https://github.com/aics-int/napari-allencell-annotator/pull/94/commits/d4a4ffff05347f903ab08b0421369f8e9a8985b8